<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipeDialog</class>
 <widget class="QDialog" name="AddRecipeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Recipe</string>
  </property>
  <widget class="QDialogButtonBox" name="addRecipeButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>1051</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="recipe_name_textedit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="new_ingredient_textedit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter ingredient text here...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recipe_description_textedit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ingredients_listwidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>691</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="instructions_listwidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>1051</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingredients:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructions:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_ingredient_pushbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_ingredient_pushbutton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addRecipeButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRecipeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRecipeButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRecipeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
